.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!--- Provide a short summary of your changes in the title above -->
```

Comment:
This bit kills me in Chef's repo... there's already obviously a short-title in the summary, and it's just a line I have to delete...